style.css

```diff
@@ -0,0 +1,683 @@
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+/* about me  */
+
+.section-3 {
+  margin-top: 100px;
+  background: #1c1a19;
+  margin-bottom: 200px;
+}
+
+.about-me {
+  display: grid;
+  grid-template-columns: auto;
+}
+
+.me {
+  margin-top: 80px;
+  position: relative;
+  width: 90%;
+  height: 79px;
+  margin-left: 5%;
+  margin-right: 5%;
+  font-family: sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 48px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  overflow: hidden;
+  color: #ff6b00;
+}
+
+.p-me {
+  position: relative;
+  width: 90%;
+  height: 148px;
+  margin-left: 5%;
+  margin-top: 40px;
+  font-family: sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+  flex: 1;
+  order: 0;
+}
+
+.back-image {
+  background-image: url(image/abtme.png);
+  background-repeat: no-repeat;
+  background-position-x: 100%;
+}
+
+.resume {
+  margin-left: 5%;
+}
+
+.line2 {
+  margin-top: 220px;
+  width: 80%;
+  margin-left: 10%;
+  color: white;
+}
+
+.lan {
+  display: inline-flex;
+  width: 60%;
+  margin-left: 20%;
+  margin-top: 100px;
+}
+
+.lan > img {
+  width: 30px;
+  height: 30px;
+  background: #ff6b00;
+  transform: rotate(-30deg);
+}
+
+.lan > h3 {
+  margin-left: 20px;
+  color: white;
+  font-family: sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 48px;
+  line-height: 44px;
+}
+
+.language > ul {
+  list-style: none;
+  color: white;
+  margin-left: 25%;
+  margin-top: 40px;
+}
+
+.language > ul > li {
+  margin-top: 40px;
+  font-family: sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 43px;
+}
+
+.frm {
+  display: inline-flex;
+  width: 100%;
+  margin-left: 10%;
+  margin-top: 100px;
+}
+
+.frm > img {
+  width: 30px;
+  height: 40px;
+}
+
+.frm > h3 {
+  margin-left: 10px;
+  color: white;
+  font-family: sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 48px;
+  line-height: 36px;
+}
+
+.framwork > ul {
+  list-style: none;
+  color: white;
+  margin-left: 25%;
+  margin-top: 40px;
+}
+
+.framwork > ul > li {
+  margin-top: 40px;
+  font-family: sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 43px;
+}
+
+.Skills {
+  margin-bottom: 200px;
+}
+
+.Skills > h3 {
+  width: 60%;
+  margin-left: 20%;
+  margin-top: 100px;
+  color: white;
+  font-family: sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 48px;
+  line-height: 44px;
+}
+
+.Skills > ul {
+  list-style: none;
+  color: white;
+  margin-left: 25%;
+  margin-top: 40px;
+}
+
+.Skills > ul > li {
+  margin-top: 40px;
+  font-family: sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 43px;
+}
+
+/* contact forms */
+
+.form-page {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  background-color: white;
+  width: 80vw;
+  height: 80vh;
+  margin: 24px 24px;
+}
```

Comment:
Your design doesn't quite match the one from the template design.

Here's your design:

<img width="190" alt="Screen Shot 2022-05-30 at 18 31 14" src="https://user-images.githubusercontent.com/82426682/171068002-29c66c54-c2f6-4626-b14b-214c44f996fa.png">

<img width="187" alt="Screen Shot 2022-05-30 at 18 32 43" src="https://user-images.githubusercontent.com/82426682/171068074-2417cee0-a3c0-495b-99e8-e4a556f57fc7.png">

Here's the template design:

<img width="271" alt="Screen Shot 2022-05-30 at 18 33 22" src="https://user-images.githubusercontent.com/82426682/171068105-e765f646-2943-4f39-836b-6a757f388b82.png">

Do you see how different they are? Kindly fix this.

Additionally, your page has a strange horizontal scrolling behavior that's probably caused by a fixed `width` in a container. Kindly fix that as well.